File: src/string/get_vowel_count.py
vowels = { "a": "a", "e": "e", "i": "i", "o": "o", "u": "u" }

def get_vowel_count(string: str):
  string = string.lower()
  count = 0
  for char in string:
    if char in vowels: count += 1
  return count
  

File: src/string/get_vowel_count.ts
const vowels = { a: "a", e: "e", i: "i", o: "o", u: "u" };

export default function getVowelCount(str: string) {
  str = str.toLowerCase();
  let count = 0;

  for (let index = 0; index < str.length; index++) {
    const char = str[index];
    if (char in vowels) count++;
  }

  return count;
}
